undefined_syms.txt

```diff
@@ -1102,23 +642,14 @@ D_06000D60 = 0x06000D60;
 
 D_08000000 = 0x08000000;
 
+// segment 0x09
+
```

Comment:
Do we need this? I don't think we'll be adding to `undefine_syms.txt`? Unless you're thinking of future versions?

Reply:
Nah it isn't needed. Personally, I like it as it shows clearly that there are no more undefined syms for segment 0x09. Without it you just have to notice the gap between segment `0x08` and `0x0A`.

Reply:
@engineer124 with regards to this, I noticed that there actually wasn't an empty segment `0x07` section, so added it.